app/main/views/suppliers.py

```diff
@@ -93,25 +93,16 @@ def view_signed_agreement(supplier_id, framework_slug):
     if not supplier_framework.get('agreementReturned'):
         abort(404)
 
-    # build an OrderedDict of applied-for lotSlug against lotName, ordered by lotSlug
     if framework["status"] in ("live", "expired"):
```

Comment:
Don't do this as a one liner (assign iterator and _then_ construct list) and DRY (OrderedDict, sorted).